app/test_app/functional/api/test_credit_transfer_api.py

```diff
@@ -50,7 +50,7 @@ def test_create_credit_transfer(test_client, authed_sempo_admin_user, create_tra
     recipient_user_id = recipient_user_id_accessor(create_transfer_account_user)
 
     if transfer_type == 'PAYMENT' and sender_user_id:
-        create_transfer_account_user.transfer_account.set_balance_offset(10000)
+        create_transfer_account_user.transfer_account.set_balance_offset(1000000)
```

Comment:
?????

Reply:
why is this off by two zero's now?

Reply:
oh it's just giving the test account more funds to play with - previous parts of the test were allowing the balance to dip into negative, where is the new code is more strict about allowing that sort of thing